index.js

```diff
@@ -111,12 +113,12 @@ Store.prototype._write = function (req) {
   })
 
   function write (store, offsets, buffers) {
-    var block
+    var block = null;
```

Comment:
undefined is fine right?

Reply:
Good catch, yes undefined is ok. This was from earlier efforts to troubleshoot issues